packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -325,22 +325,24 @@ export default class HostItem extends Component {
           </SingleLineRow>
         </BlockLink>
         {(this.state.expanded || expandAll) && (
-          <Row>
-            <Col mediumSize={2} className={styles.itemExpanded} style={{ marginTop: '0.3rem' }}>
+          <SingleLineRow>
+            <Col mediumSize={2} className={styles.itemExpanded}>
               <span>
-                {host.cpus.cores}x <Icon icon='cpu' /> &nbsp; {formatSizeShort(host.memory.size)} <Icon icon='memory' />{' '}
-                &nbsp; v{host.version.slice(0, 3)}
+                {host.cpus.cores}x <Icon icon='cpu' /> &nbsp; {formatSizeShort(host.memory.size)} <Icon icon='memory' />
               </span>
             </Col>
-            <Col mediumSize={4}>
+            <Col mediumSize={1} className={styles.itemExpanded}>
+              {host.productBrand} {host.version} <Icon icon='host' />
```

Comment:
IMO, you can even get rid of the icon to unclutter that zone as much as possible.